Whiskey/Tasks/Invoke-WhiskeyPipelineTask.ps1

```diff
@@ -0,0 +1,105 @@
+ function Invoke-PipelineTask 

```

Comment:
There is already a function named `Invoke-WhiskeyPipeline`, so I had to name this `Invoke-WhiskeyPipelineTask`. `:(`